lib/charms/layer/jenkins/packages.py

```diff
@@ -130,20 +212,18 @@ def _setup_source(self, release):
         self._apt.add_source(source, key=key)
 
     def _bundle_download(self, path=None):
-        hookenv.log(
-            "Downloading bundle from %s" % self._jc.jenkins_repo)
+        hookenv.log("Downloading bundle from %s" % self._jc.jenkins_repo)
         self._jc.get_binary_package(path)
 
     def jenkins_upgradable(self):
         """
-            Verify if there's a new version of jenkins available.
-            Note: When bundle-site is not set the return will always
-            be True.
+        Verify if there's a new version of jenkins available.
+        Note: When bundle-site is not set the return will always
+        be True.
         """
         if hookenv.config()["bundle-site"] == "":
```

Comment:
I find the code easier to read when the boolean expression is directly returned:
```
return hookenv.config()["bundle-site"] == "" or parse_version(self._jc.core_version) > parse_version(self.jenkins_version())
````

Reply:
Yes there is a lot that could be improved here, we're considering the future of this charm (including rebuild) so let's hold off on these kind of changes to future PRs

Reply:
Works for me